src/Schedule/Schedule.js

```diff
@@ -0,0 +1,39 @@
+import './Schedule.scss'
+
+function Schedule() {
```

Comment:
Probably a stretch goal / future iteration to have a functional schedule for appointments or staffing without just hardcoding it. 
But I think it's nice to have there, and we can just hard code for now to be an example.